probe/probe.go

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package probe
+
+
+import (
+	"fmt"
+	"syscall"
+    "strings"
+	"golang.org/x/mod/semver"
+    "io/ioutil"
+    "log"
+
+)
+
+func PrintProbeResult() error{
+    
+    fmt.Printf("Host: \n")
+    fmt.Printf("\t Observability/Audit: %s \n", checkHostAuditSupport())
+    fmt.Printf("\t Enforcement: %s \n", checkHostLsmSupport())
+	return nil
+}
+
+func checkHostAuditSupport() string {
+    hostKernelVersion, err := detectHostKernelVersion()
+	if err != nil {
+		return ""
```

Comment:
```suggestion
		return "Error"
```
to let the users know that we had an Error while trying to Probe

Reply:
well noted